<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Arma_Class_parser_maingui</class>
 <widget class="QMainWindow" name="Arma_Class_parser_maingui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1609</width>
    <height>1117</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Arma Class parser Utility by Giddi</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Search DB</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0" alignment="Qt::AlignRight">
         <widget class="QLabel" name="cfg_label">
          <property name="text">
           <string>Cfg:</string>
          </property>
         </widget>
        </item>
        <item row="6" column="2">
         <widget class="QGroupBox" name="notelist_groupBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string>Note list</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="2" column="0">
            <widget class="QListWidget" name="actual_notlist_listWidget">
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QPushButton" name="export_notelist_pushButton">
             <property name="text">
              <string>Export</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0" alignment="Qt::AlignHCenter">
            <widget class="QToolButton" name="configure_note_list_toolButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>1</horstretch>
               <verstretch>1</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>176</width>
               <height>0</height>
              </size>
             </property>
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLineEdit" name="notelist_name_lineEdit"/>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="notelist_label">
             <property name="text">
              <string>Name(optional):</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QGroupBox" name="name_options_groupBox">
          <property name="title">
           <string>Autocomplete</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QRadioButton" name="use_classnames_radioButton">
             <property name="text">
              <string>Use Class names</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="use_displaynames_radioButton">
             <property name="text">
              <string>Use Display names</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="4" column="1" rowspan="3">
         <widget class="QTableView" name="output_tableView">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Verdana Pro</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="frameShape">
           <enum>QFrame::WinPanel</enum>
          </property>
          <property name="lineWidth">
           <number>4</number>
          </property>
          <property name="midLineWidth">
           <number>2</number>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QAbstractScrollArea::AdjustToContents</enum>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="textElideMode">
           <enum>Qt::ElideMiddle</enum>
          </property>
          <property name="gridStyle">
           <enum>Qt::DashDotLine</enum>
          </property>
          <attribute name="horizontalHeaderCascadingSectionResizes">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderMinimumSectionSize">
           <number>50</number>
          </attribute>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>200</number>
          </attribute>
         </widget>
        </item>
        <item row="6" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="cfg_selection_search_comboBox">
          <property name="maximumSize">
           <size>
            <width>250</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="3">
         <widget class="QGroupBox" name="search_input_groupBox">
          <property name="title">
           <string/>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="2" column="0">
            <widget class="QLineEdit" name="search_input_lineEdit"/>
           </item>
           <item row="2" column="2">
            <widget class="QComboBox" name="search_comboinput_comboBox">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>250</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="1" column="1" rowspan="2">
            <widget class="Line" name="line">
             <property name="frameShadow">
              <enum>QFrame::Plain</enum>
             </property>
             <property name="lineWidth">
              <number>2</number>
             </property>
             <property name="midLineWidth">
              <number>2</number>
             </property>
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QRadioButton" name="use_lineinput_radioButton">
             <property name="text">
              <string>Search from text</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QCheckBox" name="only_with_images_checkBox">
          <property name="text">
           <string> only Entries with images</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Config Tree</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="1">
         <widget class="QPushButton" name="show_data_tree_pushButton">
          <property name="text">
           <string>show Tree</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>350</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="0" colspan="3">
         <widget class="DataTreeWidget" name="plotting_DataTreeWidget"/>
        </item>
        <item row="1" column="0">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>350</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1609</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatabase">
    <property name="title">
     <string>Database</string>
    </property>
    <addaction name="actioninitialize_Database"/>
    <addaction name="actionSelect_new_Config_Dumb_File"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionopen_Configurator"/>
   </widget>
   <addaction name="menuDatabase"/>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget_2">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>1</horstretch>
     <verstretch>1</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>274</width>
     <height>297</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>2048</width>
     <height>2048</height>
    </size>
   </property>
   <property name="windowIcon">
    <iconset resource="Arma_class_parser_ressources.qrc">
     <normaloff>:/icons/ressources/icons/gid_logo_temp.ico</normaloff>:/icons/ressources/icons/gid_logo_temp.ico</iconset>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Image Viewer</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>1</horstretch>
      <verstretch>1</verstretch>
     </sizepolicy>
    </property>
    <layout class="QGridLayout" name="gridLayout_7">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="picture_label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>2</horstretch>
         <verstretch>2</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>256</width>
         <height>256</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>2048</width>
         <height>2048</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>1</width>
         <height>1</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>256</width>
         <height>256</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="lineWidth">
        <number>2</number>
       </property>
       <property name="midLineWidth">
        <number>2</number>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="Arma_class_parser_ressources.qrc">:/icons/ressources/icons/gid_logo_temp.ico</pixmap>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionopen_Configurator">
   <property name="text">
    <string>open Configurator</string>
   </property>
  </action>
  <action name="actioninitialize_Database">
   <property name="text">
    <string>initialize Database</string>
   </property>
  </action>
  <action name="actionSelect_new_Config_Dumb_File">
   <property name="text">
    <string>Select new Config Dumb File</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DataTreeWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Arma_class_parser_ressources.qrc"/>
 </resources>
 <connections/>
</ui>
